src/js/player.js

```diff
@@ -222,10 +222,17 @@ class Player extends Component {
     // Default state of video is paused
     this.addClass('vjs-paused');
 
-    // Make box use width/height of tag, or rely on default implementation
-    // Enforce with CSS since width/height attrs don't work on divs
-    this.width(this.options_['width'], true); // (true) Skip resize listener on load
-    this.height(this.options_['height'], true);
+    // Add a style element in the player that we'll use to set the width/height
+    // of the player in a way that's still overrideable by CSS, just like the
+    // video element
+    this.styleEl_ = document.createElement('style');
+    el.appendChild(this.styleEl_);
```

Comment:
This will come after the `head`, where most people will be setting their styles. So this sort of rule:
```css
.video-js {
  width: 100%;
}
```
would not override any rules in the base styles.